scanner/src/main/kotlin/scanners/fossid/FossId.kt

```diff
@@ -358,30 +366,34 @@ class FossId internal constructor(
         revision: String,
```

Comment:
Could you please add documentation to this function ?
Returning two values adds complexity and one cannot see straightaway what this function returns without the documentation.